examples/docs/en-US/input-number.md

```diff
@@ -69,6 +69,26 @@ Allows you to define incremental steps.
 ```
 :::
 
+### Multiple of Step
+
+:::demo The `multiple-of-step` attribute accepts a `boolean`. if this attribute is `true`, input value can not be anything but multiple of step.
```

Comment:
how about: "input value can only be multiples of `step`."

Reply:
This is better, thanks